src/sgame/sg_cmds.cpp

```diff
@@ -3497,6 +3497,116 @@ void Cmd_Reload_f( gentity_t *ent )
 	}
 }
 
+/*
+=================
+Cmd_TeamStatus_f
+=================
+*/
+void Cmd_TeamStatus_f(gentity_t * ent)
+{
+	char multiomrc[12];
+	int builders = 0;
+	int arm = 0, mediboost = 0, miner = 0;
```

Comment:
I would rather see an array of ints like I did in #1973, it makes things easier to extend, and requires less individual variables.